### DISCUSSION TITLE
blog/working-with-aws-s3-using-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#37] [General]
# blog/working-with-aws-s3-using-aspnet-core/

We will get started on working with AWS S3 using ASP.NET Core Web API to upload, download and delete files from Amazon's Super Scalable S3!

https://codewithmukesh.com/blog/working-with-aws-s3-using-aspnet-core/

**Comment by alexovonathan22** [comment]
Hello Mukesh,
Thanks for this,

I have a challenge, the is implementation works locally and i see the image uploaded but when deployed to aws lambda the object url i get returns a broken image,
